src/input.js

```diff
@@ -359,15 +359,16 @@ editHandlers.input = view => DOMChange.start(view)
 function captureCopy(view, dom) {
   // The extra wrapper is somehow necessary on IE/Edge to prevent the
   // content from being mangled when it is put onto the clipboard
-  let wrap = document.body.appendChild(document.createElement("div"))
+  let doc = view.root
```

Comment:
`view.root` may be a shadow root, which has neither `body` nor `createElement` methods. Is there any problem with using `ownerDocument` here as well?